bundles/org.openhab.ui/web/src/components/widgets/layout/oh-grid-item.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <grid-item v-bind="$attrs" v-if="visible" class="oh-grid-item card no-margin" @moved="movedEvent" @resized="resizedEvent" dragAllowFrom=".drag-handle">
+    <template v-if="context.editmode">
+      <f7-link :popover-open="'.item-popover-' + _uid" class="configure-item-menu" ><f7-icon f7="menu"></f7-icon></f7-link>
+      <f7-popover :class="'item-popover-' + _uid">
+        <f7-list>
+          <f7-list-item v-if="context.component.slots.default.length > 0" title="Configure Widget" @click="context.editmode.configureWidget(context.component.slots.default[0], context)" link="#" popover-close no-chevron><f7-icon slot="media" f7="gear_alt" /></f7-list-item>
+          <f7-list-item v-if="context.component.slots.default.length > 0" title="Edit YAML" @click="context.editmode.editWidgetCode(context.component.slots.default[0], context)" link="#" popover-close no-chevron><f7-icon slot="media" f7="square_pencil" /></f7-list-item>
+          <f7-list-item title="Remove Item" @click="context.editmode.removeWidget(context.component, context.parent, 'grid')" link="#" popover-close no-chevron><f7-icon slot="media" f7="xmark_circle" /></f7-list-item>
+        </f7-list>
+      </f7-popover>
```

Comment:
```suggestion
      <f7-popover :class="'item-popover-' + _uid" :backdrop="false">
        <div class="display-flex margin justify-content-center">
          <f7-link href="#" v-if="context.component.slots.default.length > 0" class="text-color-blue display-flex flex-direction-column margin-right" :popover-close="'.item-popover-' + _uid" @click="context.editmode.configureWidget(context.component.slots.default[0], context)" icon-f7="square_pencil">Configure</f7-link>
          <f7-link href="#" v-if="context.component.slots.default.length > 0" class="text-color-blue display-flex flex-direction-column margin-right" :popover-close="'.item-popover-' + _uid" @click="context.editmode.editWidgetCode(context.component.slots.default[0], context)" icon-f7="doc_text">YAML</f7-link>
          <!-- <f7-link href="#" class="text-color-blue display-flex flex-direction-column margin-right" @click="context.editmode.copyWidget(context.component, context.parent)" icon-f7="scissors_alt">Copy</f7-link> -->
          <f7-link href="#" class="text-color-red display-flex flex-direction-column" :popover-close="'.item-popover-' + _uid" @click="context.editmode.removeWidget(context.component, context.parent, 'grid')" icon-f7="trash">Remove</f7-link>
        </div>
      </f7-popover>
```

It makes the popover of like the one on the floor plan pages, it's better for consistency sake I believe.
This needs some work, but it would be even better if the drop shadow and "angle" (callout) would resemble those (overriding CSS variables maybe, and additional classes). Leaving out the "copy" link because there's no paste.

![image](https://user-images.githubusercontent.com/2004147/109338392-7fbca980-7866-11eb-853c-3bb55ada4a00.png)

Reply:
```
    background-color: #fff;
    border-radius: 12px;
    box-shadow: 0 3px 14px rgb(0 0 0 / 40%);
    will-change: transform, opacity;
    font: 12px/1.5 Helvetica Neue,Arial,Helvetica,sans-serif;
```
on the popover makes it look quite similar (at least in the desktop theme).